### DISCUSSION TITLE
AdEMAMix experimentation

### DISCUSSION THREAD

**Discussion by ExtReMLapin** [ultralytics/ultralytics#16072] [Show and tell]
https://arxiv.org/abs/2409.03137

![image](https://github.com/user-attachments/assets/1103fb7f-7bd9-4caf-af6b-ffa71c1062af)

![image](https://github.com/user-attachments/assets/e44aa43b-c866-4fac-ad50-df4458292917)

**Comment by ExtReMLapin** [comment]
erh...

![image](https://github.com/user-attachments/assets/2602f2ad-28b1-452c-9b62-17bd31a189b8)

**Reply by Burhan-Q** [reply]
Which one uses the `AdEMAMix` Optimizer? I saw this publication and was wondering if it would be beneficial for YOLO
